src/runtime/server/preact_hooks.tsx

```diff
@@ -474,9 +474,7 @@ function FreshRuntimeScript() {
       .join(",") +
       "}";
 
-    const serializedProps = JSON.stringify(
-      stringify(islandProps, stringifiers),
-    );
+    const serializedProps = stringify(islandProps, stringifiers);
```

Comment:
`stringify()` returns a string. `JSON.stringify(string)` returns that string. So returns the same value.